common/changes/@stonecrop/beam/feat-beam-list-filter-style_2024-11-04-03-12.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@stonecrop/beam",
+      "comment": "added BeamFilter component",
+      "type": "none"
```

I don't have a good suggestion to automate this, but this should be changed to the following for generating changelogs correctly.

```suggestion
      "type": "patch"
```